### PR TITLE
Minor improvement on wording for sample

### PR DESCRIPTION
**Pull request by niemela**

### CODE REVIEW

File: spec/2023-07-draft.md [hunk 1]
```diff
@@ -627,7 +627,7 @@ Also, it does not _have to_ contain any test cases, for example when the samples
 By default, the `.in` and `.ans` pairs from `data/sample` are shown in the problem statement.
 This can be customized by creating the `data/sample/statement` directory.
 If this directory exists, _only_ those testcases present in this directory are shown,
-and the test cases in `data/sample` are ignored for purposes of the problem statement.
+and the test cases in `data/sample` are **not** included in the problem statement.
```
Comment (RagnarGrootKoerkamp) [inline spec/2023-07-draft.md:630]: well, unless the ones in `data/sample/statement` are the same...

Maybe just drop the entire `, and the test cases in `data/sample` are ignored` part?

Comment (eldering) [reply spec/2023-07-draft.md:630]: What about:
If this directory exists, its contents replaces that of `data/sample` for purposes of the problem statement.

File: spec/2023-07-draft.md [hunk 2]
```diff
@@ -627,7 +627,7 @@ Also, it does not _have to_ contain any test cases, for example when the samples
 By default, the `.in` and `.ans` pairs from `data/sample` are shown in the problem statement.
 This can be customized by creating the `data/sample/statement` directory.
 If this directory exists, _only_ those testcases present in this directory are shown,
```
Comment (niemela) [inline spec/2023-07-draft.md:629]:
```suggestion
If this directory exists, its contents replaces that of `data/sample` for purposes of the problem statement.
```

File: spec/2023-07-draft.md (hunk 1 above)
Comment (niemela) [inline spec/2023-07-draft.md:630]:
```suggestion
```

**Pull request merged**